test/expr.jl

```diff
@@ -0,0 +1,538 @@
+macro audit(expr::Expr)
```

Comment:
A comment saying what this does would be nice.

Reply:
I added a docstring